wordpress-mobile/WordPress-iOS
# NUX button hotfix

Fixes # bug on 14.5 build

### To test:
- check out this branch
- `rake dependencies`
- build and run
- log out
- log in via WordPress.com
- note that all the "pink" buttons are the correct size

| Before | After |
| --- | --- | 
| <img src="https://user-images.githubusercontent.com/1062444/77328790-0a0d9c80-6ceb-11ea-971f-acbb25c45f80.PNG" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/77328811-109c1400-6ceb-11ea-89cd-b39da840579f.png" width="350" /> |

### Do not merge until:
1. Authenticator PR merged https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/216 ✔️
2. Authenticator 1.11.1-beta.1 published to Cocoapods ✔️
3. This branch points to the updated Auth release ✔️

PR submission checklist:
